doc/theory/glossary.tex

```diff
@@ -1,12 +1,75 @@
-\documentclass{article}
-\usepackage{amssymb}
+\documentclass{amsart}
+\usepackage{amsmath, amssymb, amsfonts}
+\usepackage{enumerate}
+\usepackage{tikz-cd}
+\usepackage{bm}
+%\usepackage[bookmarks=true,
```

Comment:
I'd suggest removing commented-out lines altogether, unless uncommenting them can be important for some users?